lumicks/pylake/correlated_stack.py

```diff
@@ -291,14 +308,21 @@ def plot_correlated(
         title_factory = lambda frame: make_image_title(self, frame, show_name=False)
         frame_timestamps = self.frame_timestamp_ranges
 
+        def frame_grabber(frame_idx):
+            return self._get_frame(frame_idx)._get_plot_data(channel, adjustment=adjustment)
+
+        def post_update(image_handle, image):
+            return adjustment._update_limits(image_handle, image, channel)
```

Comment:
Can this be wrapped into the base `plot_correlated` function somehow? Because right now both here and for `Scan` you have the exact same `post_udpate` defined.

Reply:
I did this intentionally. I don't want `plot_correlated` to be dependent on implementational details of `ColorAdjustment`. `plot_correlated` may be used for other things in the future as well. I don't really think deduplicating 1 line of code is worth coupling them?

Reply:
Yeah ok, fair enough. In that case you might want to consider pulling out the `post_update` in `Scan.plot_correlated()` to match what you do here. Defining that lambda in the function call as an argument looks a bit strange the way it's formatted, but really that's just being super nitpicky so feel free to just keep it as is

Reply:
Okay. I'll pull it out :)